tests/Eccube/Tests/Web/Admin/Setting/Shop/DeliveryControllerTest.php

```diff
@@ -0,0 +1,261 @@
+<?php
+/*
+ * This file is part of EC-CUBE
+ *
+ * Copyright(c) 2000-2015 LOCKON CO.,LTD. All Rights Reserved.
+ *
+ * http://www.lockon.co.jp/
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.
+ */
+
+
+namespace Eccube\Tests\Web\Admin\Setting\Shop;
+
+use Eccube\Common\Constant;
+use Eccube\Entity\PaymentOption;
+use Eccube\Tests\Web\Admin\AbstractAdminWebTestCase;
+
+class DeliveryControllerTest extends AbstractAdminWebTestCase
+{
+    public function createDelivery()
```

Comment:
Proposal use of [Fixutre\Generator::createDelivery](https://github.com/EC-CUBE/ec-cube/blob/f4014121bc0fc88cf3ef7d5d8ad9397a27808b5d/tests/Eccube/Tests/Fixture/Generator.php#L453)